docs/contributor-guide/developer-guide/database/db-schema.md

```diff
@@ -1,6 +1,6 @@
 # DB Schema
 
-The database schema is derived by combining all of the [evolution SQL files](https://github.com/seattle-uat/civiform/tree/main/server/conf/evolutions/default) in sequential order. It's current state as of #36 follows.  This can be generated using `pg_dump -s postgres` and then `\d tableName` on the table of interest.
+The database schema is derived by combining all of the [evolution SQL files](https://github.com/seattle-uat/civiform/tree/main/server/conf/evolutions/default) in sequential order. It's current state as of #40 follows.  This can be generated using `pg_dump -s postgres` and then `\d tableName` on the table of interest.
```

Comment:
```suggestion
The database schema is derived by combining all of the [evolution SQL files](https://github.com/seattle-uat/civiform/tree/main/server/conf/evolutions/default) in sequential order. Its current state as of #40 follows.  This can be generated using `pg_dump -s postgres` and then `\d tableName` on the table of interest.
```